### DISCUSSION TITLE
Relation `fromModel` custom name

### DISCUSSION THREAD

**Discussion by m-pastuszek** [orchidsoftware/platform#2752] [Q&A] [answered]
Hello, I'd like to show combination of two columns in user-visible select name in Relation field-type. As it's easy for TD and Sight using `render()`, I don't know how can I make the same for `Relation` field, but I can type only one column as second parameter in `fromModel()` function.

My case is following:
I have select Workplace field. I'd like show user available options as `Department - WorkplaceName`.

TD example:
```php
 TD::make('workplace', 'Workplace')
                ->render(function ($model) {
                    return ($model->workplace->department->name . ' - ' . $model->workplace->name);
                }),
```

Any ideas? :)

**Comment by tabuna** [comment, marked as answer]
You can add a calculated attribute to your model, for example

```php
class Workplace extends Model
{
    /**
     * @return string
     */
    public function getFullNameAttribute(): string
    {
        return $model->department->name . ' - ' . $model->name;
    }
}
```

Then use the `displayAppend` method to specify which calculated field to use for display:

```php
Relation::make('workplace.')
    ->fromModel(Workplace::class, 'name')
    ->displayAppend('fullName')
    ->multiple()
    ->title('Select workplace');
```

**Reply by m-pastuszek** [reply]
Ahhh... so easy!
Thank you so much :)
